packages/server/src/unstable-core-do-not-import/parser.ts

```diff
@@ -110,5 +118,25 @@ export function getParseFn<TType>(procedureParser: Parser): ParseFn<TType> {
     };
   }
 
+  if ('~standard' in parser) {
```

Comment:
It would probably be sensible to move this higher up. All of the existing validation libraries won't actually reach here. 

Now that standard-schema exists, the onus should be on validation libraries to implement it, and trpc's docs should slowly move towards "just use any standard-schema lib". At that point, most users will expect the `parser['~standard'].validate(...)` method to be used over `parser.parse(value)` or whatever. Hopefully there being any differences would be very rare, but still I think better to prioritise standard-schema.

Reply:
that would be a breaking change though. since for example a validation failure on a zod schema would not throw ZodError when using `~standard.parse`

Reply:
That's why I put it at the end. Once Standard Schema is more established, tRPC may move it to the top when they release a new version.